@storybook/react.tsx

```diff
@@ -1,3 +1,4 @@
+//@ts-ignore
```

Comment:
upgrading typescript causes this to error downstream. addon-info hasn't been migrated to typescript yet: https://github.com/storybookjs/storybook/issues/5030